src/meta/size.rs

```diff
@@ -2,10 +2,14 @@ use crate::color::{ColoredString, Colors, Elem};
 use crate::flags::{Flags, SizeFlag};
 use std::fs::Metadata;
 
-const KB: u64 = 1024;
-const MB: u64 = 1024_u64.pow(2);
-const GB: u64 = 1024_u64.pow(3);
-const TB: u64 = 1024_u64.pow(4);
+const KIB: u64 = 1024;
```

Comment:
can you add a comment here to record that we use `KIB` instead of `KiB` for rust naming conventions?